ragen/env/search/utils/reward_score/qa_em.py

```
@@ -0,0 +1,139 @@
+    matches = list(match)
+    
+    # If there are 0 or exactly 1 matches, return None
+    # This is intentional: EM scoring expects proper format with reasoning + answer blocks
+    if len(matches) <= 1:
+        return None
+    
+    # If there are 2 or more matches, return the last one
+    return matches[-1].group(1).strip()
+
+
+def compute_score_em(solution_str, ground_truth, method='strict', format_score=0., score=1.):
```

Ruff (0.14.13)

Unused function argument: `method`

(ARG001)